src/js/HRModules/addUserToForm.js

```diff
@@ -0,0 +1,110 @@
+
+const { $id, $ } = require('../utils');
+
+const AddUserToForm = {
+    initialized: false,
+    chosen: null,
+
+    init() {
+        if (this.initialized)
+            return;
+
+        this.initialized = true;
+
+        this.chooseSection(0);
+    },
+
+    open() {
+        if (!this.initialized)
+            this.init();
+
+        this.chooseSection(0);
+
+        const users = $id('addUserToForm-users').children;
+        for (const el of users)
+            el.querySelectorAll('input')[0].checked = false;
+
+        const forms = $id('addUserToForm-forms').children;
+        for (const el of forms)
+            el.querySelectorAll('input')[0].checked = false;
+    },
+
+    chooseSection(no) {
+        if (this.chosen === no)
+            return;
+
+        if (no === 2)
+            this.validate();
+
+        const els = $('.addUserToForm-headerPost');
+
+        if (this.chosen !== null)
```

Comment:
A może !this.chosen?

Reply:
nope, bo this.chosen może być 0 czyli nie przejdzie